packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/resizingModule.tests.js

```diff
@@ -131,6 +131,34 @@ module('Resizing module', moduleConfig, () => {
         assert.deepEqual(resizingInstance.allowedTargets, ['video'], '\'allowedTargets\' option has been applied');
     });
 
+    QUnit.module('on runtime change', () => {
+        test('apply enabled option as a object on runtime change and still render resize frame (T1241439)', function(assert) {
+            const resizingInstance = new Resizing(this.quillMock, this.options);
+            this.attachSpies(resizingInstance);
+            resizingInstance.option('mediaResizing', { enabled: true });
+
+            assert.ok(resizingInstance.enabled, '\'enabled\' option has been applied');
+            assert.strictEqual(this.$element.find(`.${RESIZE_FRAME_CLASS}`).length, 1, 'There is resize frame element');
+            assert.ok(this.attachEventsSpy.calledOnce, 'events has been attached');
+            assert.ok(this.detachEventsSpy.notCalled, 'events hasn\'t detached');
+        });
+
+        test('Ensure the \'enabled\' option can be disabled at runtime and correctly removes the resize frame (T1241439)', function(assert) {
+            const resizingInstance = new Resizing(this.quillMock, this.options);
+            this.attachSpies(resizingInstance);
+
+            resizingInstance.option('mediaResizing', { enabled: true });
+            assert.ok(resizingInstance.enabled, '\'enabled\' option should be active');
+            assert.ok(this.attachEventsSpy.calledOnce, 'Events should be attached once.');
```

Comment:
This test is about runtime change to `{ enabled: false }`, let's not add actions and asserts for other runtime changes here - test should be atomic.

It's better to change initial prop value - just set `this.option.enabled = true` before creating of a Resizing instance